modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -0,0 +1,49 @@
+= Get Started with Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, installation prerequisites
+
+image::prepare-to-install-breadcrumb.png[]
+
+To start using Anypoint Service Mesh, you must have installed specific applications and must have specific roles and permissions. This section describes the following requirements:
```

Comment:
@priyaayyar which application is it referring to?
> you must have installed specific applications

Reply:
updated the section.